examples/oft-aptos/tasks/utils/utils.ts

```diff
@@ -0,0 +1,313 @@
+import hardhatConfig from '../../hardhat.config'
```

Comment:
you can't import hardhat config into typescript script like that

initOFT script is broken now:
```
➜  oft-aptos git:(aptos-cli) ✗ bun tasks/move/initAptosOFT.ts
31 |     static getHardhatContext() {
32 |         const globalWithHardhatContext = global;
33 |         const ctx = globalWithHardhatContext.__hardhatContext;
34 |         if (ctx === undefined) {
35 |             throw new errors_1.HardhatError(errors_list_1.ERRORS.GENERAL.CONTEXT_NOT_CREATED);
36 |         }
                   ^
HardhatError: HH5: HardhatContext is not created.
      at getHardhatContext (/Users/danielkmak/projects/devtools/node_modules/.pnpm/hardhat@2.22.12_ts-node@10.9.2_typescript@5.5.3/node_modules/hardhat/internal/context.js:36:15)
      at extendEnvironment (/Users/danielkmak/projects/devtools/node_modules/.pnpm/hardhat@2.22.12_ts-node@10.9.2_typescript@5.5.3/node_modules/hardhat/internal/core/config/config-env.js:63:1)
      at /Users/danielkmak/projects/devtools/node_modules/.pnpm/@nomiclabs+hardhat-ethers@2.2.3_ethers@5.7.2_hardhat@2.22.12/node_modules/@nomiclabs/hardhat-ethers/internal/index.js:14:5
      at native:1:1
```

Reply:
most likely `bun` runtime being weird since it does not have 100% compliancy with npm packages